Extensions/EPPlusExtensions.cs

```diff
@@ -4,12 +4,14 @@ namespace shipping_service.Extensions
 {
     public static class EPPlusExtensions
     {
-
         public static int GetColumnByName(this ExcelWorksheet ws, string columnName)
         {
-            if (ws == null) throw new ArgumentNullException(nameof(ws));
+            if (ws == null)
```

Comment:
Consider what autoformating has done in SeedData.cs and this. These are not consistent.